<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1643</width>
    <height>1014</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FIT4-5仿真平台</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>810</y>
      <width>451</width>
      <height>161</height>
     </rect>
    </property>
    <widget class="QComboBox" name="UserComboBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>115</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:6px;
padding:2px 4px;
border-style: outset;
border:2px groove gray;
</string>
     </property>
     <item>
      <property name="text">
       <string>WenDong</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mr.Liu</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>GangHui</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>LanJun</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="userhead">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>110</width>
       <height>110</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:6px;
padding:2px 4px;
border-style: outset;
border:2px groove gray;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTextEdit" name="chat_text">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>0</y>
       <width>321</width>
       <height>150</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:6px;
padding:2px 4px;
border-style: outset;
border:2px groove gray;
background-color: rgb(251, 251, 251);
border-top-color:rgb(186, 189, 182);
border:none;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Send_Button">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>110</y>
       <width>75</width>
       <height>35</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:6px;
padding:2px 4px;
border-style: outset;
border:2px groove gray;
border:none;
background-color: rgb(229, 234, 233);
color: rgb(37, 194, 118);
</string>
     </property>
     <property name="text">
      <string>发送(↵)</string>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>0</y>
      <width>911</width>
      <height>971</height>
     </rect>
    </property>
    <widget class="QLabel" name="map">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>910</width>
       <height>910</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:6px;
padding:2px 4px;
border-style: outset;
border:2px groove gray;</string>
     </property>
     <property name="locale">
      <locale language="Chinese" country="China"/>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="cleartrackbutton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>930</y>
       <width>910</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:6px;
padding:2px 4px;
border-style: outset;
border:2px groove gray;
background-color: rgb(229, 234, 233);
color: rgb(37, 194, 118);</string>
     </property>
     <property name="text">
      <string>Clear the Track</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>1370</x>
      <y>10</y>
      <width>271</width>
      <height>961</height>
     </rect>
    </property>
    <widget class="QTextBrowser" name="chat_interface">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>251</width>
       <height>951</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="tabChangesFocus">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>441</width>
      <height>791</height>
     </rect>
    </property>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>441</width>
       <height>791</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QListWidget{background-color: rgb(247, 247, 247); color:rgb(51,51,51); border: 1px solid  rgb(247, 247, 247);outline:0px;}
QListWidget::Item{background-color: rgb(247, 247, 247);}
QListWidget::Item:hover{background-color: rgb(247, 247, 247); }
QListWidget::item:selected{
	background-color: rgb(247, 247, 247);
	color:black; 
    border: 1px solid  rgb(247, 247, 247);
}
QListWidget::item:selected:!active{border: 1px solid  rgb(247, 247, 247); background-color: rgb(247, 247, 247); color:rgb(51,51,51); } </string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1643</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_open_map"/>
    <addaction name="action_exit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_open_map">
   <property name="text">
    <string>&amp;open_map</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>&amp;exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
